compiler/semtypes.nim

```diff
@@ -40,6 +40,12 @@ const
   errNoGenericParamsAllowedForX = "no generic parameters allowed for $1"
   errInOutFlagNotExtern = "the '$1' modifier can be used only with imported types"
 
+
+proc isDiscardUnderscoreV(v: PSym): bool =
+  if v.name.s == "_":
+    v.flags.incl(sfGenSym)
```

Comment:
Don't hide mutation in a proc with an `isX` name. I would just inline this proc.